<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MdccWindow</class>
 <widget class="QMainWindow" name="MdccWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MDCC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Commands</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="PBMdccPhysic">
         <property name="text">
          <string>Physic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PBMdccResume">
         <property name="text">
          <string>Resume</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PBMdccPause">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PBMdccResetTdc">
         <property name="text">
          <string>Reset TDC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PBMdccResetCounters">
         <property name="text">
          <string>Reset Counters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PBMdccSet">
         <property name="text">
          <string>Set Registers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PBStatus">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Registers</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QFrame" name="frame_5">
         <property name="minimumSize">
          <size>
           <width>181</width>
           <height>61</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Window Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SBWindowMode">
            <property name="minimumSize">
             <size>
              <width>57</width>
              <height>14</height>
             </size>
            </property>
            <property name="toolTip">
             <string extracomment="WindowConfig_reg(5 downto 0)=“000001” –&gt; start of spill et end of spill&#9; WindowConfig_reg(5 downto 0)=“000010” –&gt; start of spill et duree programmable WindowConfig_reg(5 downto 0)=“000100” –&gt; compteur interne WindowConfig_reg(5 downto 0)=“001000” –&gt; calibration avec spillon/spilloff interne declenché par calib_reg(1),  ne devrait pas fournir de fenetres courtes (comme spioll interne asynchrone avec calib_reg  WindowConfig_reg(5 downto 0)=“010000” –&gt; compteur interne mais seulement sur start_of_spill et duree programmable WindowConfig_reg(5 downto 0)=“100000” –&gt; compteur interne avec debut comptage sur finbusy"/>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>256</number>
            </property>
            <property name="value">
             <number>64</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Spill On</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LESpillOn">
            <property name="text">
             <string>10000</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Spill Off</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LESpillOff">
            <property name="text">
             <string>10000</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Busy Latch</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SBBusy">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Beam</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LEBeamOn">
            <property name="text">
             <string>10000000</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>221</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="tbSTATUS"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
